todoman/ui.py

```diff
@@ -251,9 +251,23 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        * if no date is supplied, it returns empty_date
+        """
         if date:
-            rv = date.strftime(self.date_format)
-            return rv
+            date_tomorrow = self.now.date() + timedelta(days=1)
+            # Get a dictionary which stores the dates for which
+            # we don't have to merely return a date string and
+            # map it to the special string we need to return
+            WIDTH = len(date.strftime(self.date_format))
```

Comment:
Ugh, you're completely right, I missed that! I think we need to rewrite how we calculate `empty_date` and avoid that trap:

```python
empty_date = self.now.format(self.date_format)
```

I don't think we need the comment explaining what that dictionary is; it seems quite clear. Maybe just a short one-liner, at most.